draft-ietf-scitt-architecture.md

```diff
@@ -619,6 +619,39 @@ For example, consider a Transparency Service implemented using a set of replicas
 Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
 This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
 
+### Configuration 
+
+The Transparency Service records its configuration in the Append-Only Log using Transparent Statements with distinguished media type `application/x-scitt-configuration`. 
```

Comment:
is this the `3 payload type` in the protected header?
Agree we need a way to determine this. I do wonder how the payload type and mediaType that says it's json, XML, rdf, or others would conflict with this.